doc/tutorials/plutus-tx.rst

```diff
@@ -6,53 +6,46 @@ What is Plutus Tx?
 
 Plutus applications are written as a single Haskell program, which
 describes both the code that runs off the chain (on a user’s computer,
-or in their wallet), and on the chain (as part of transaction
-validation).
+or in their wallet, for example), and on the chain as part of transaction
+validation.
 
 The parts of the program that describe the on-chain code are still just
-Haskell, but they are compiled into :term:`Plutus Core` (instead of into the
-normal compilation target language) and we refer to them as :term:`Plutus Tx`
-("Tx" since they usually go into transactions).
+Haskell, but they are compiled into :term:`Plutus Core`, rather than into 
+the normal compilation target language. We refer to them as :term:`Plutus Tx`
+(where Tx indicates that these components usually go into transactions).
 
-.. warning::
+.. **warning::
 
-   Strictly speaking, only a subset of Haskell is supported inside
-   Plutus Tx blocks, but the majority of simple Haskell will work, and
-   the Plutus Tx compiler will tell you if you use something that is
-   unsupported.
+   Strictly speaking, while the majority of simple Haskell will work, only a 
+   subset of Haskell is supported inside Plutus Tx blocks. The Plutus Tx compiler
+   will tell you if you are attempting to use an unsupported component.
 
 The key technique that we use to implement Plutus Tx is called *staged
-metaprogramming*. What that means is that the main Haskell program
-generates *another* program, in this case the Plutus Core program that
-will run on the blockchain. Plutus Tx is the mechanism that we use to
-write those programs. But because Plutus Tx is just part of the main
-Haskell program we can share types and definitions between the two.
+metaprogramming*, which means that the main Haskell program generates *another* program 
+(in this case, the Plutus Core program that will run on the blockchain). Plutus Tx is the 
+mechanism we use to write those programs, but since Plutus Tx is just part of the main
+Haskell program, we can share types and definitions between the two.
```

Comment:
I'd suggest not reflowing the text if you can avoid it, it makes it harder to review changes...

(I've actually started writing with one sentence per line these days for that reason, but that's not to everyone's taste!)